tests/backend.js

```diff
@@ -0,0 +1,24 @@
+var test = require('tape');
+var search = require("../src/search.js");
+
+// test RegExp
+test("regEx should return words with the entered start letters", function(t) {
+  var testString = "james, minesh, dak, jen"
+  var query = 'ja';
+  var regEx = new RegExp('\\b(' + query + ')\\w*', 'gi');
```

Comment:
The tests don't use our search function - they replicate the code in the function but it might be better to actually use the function itself